### ISSUE TITLE
Multiple trees are plotted by shape instead of label 

### ISSUE THREAD

**Issue by richelbilderbeek** [GuangchuangYu/ggtree#221]
### Prerequisites

+ [x] Have you read [Feedback](https://guangchuangyu.github.io/ggtree/#feedback) and follow the [guide](https://guangchuangyu.github.io/2016/07/how-to-bug-author/)?
	* [x] make sure your are using the latest release version
	* [x] read the [documents](https://guangchuangyu.github.io/ggtree/documentation/)
	* [x] google your quesion/issue

### Describe you issue

* [x] Make a reproducible example (*e.g.* [1](https://gist.github.com/talonsensei/e1fad082657054207f249ec98f0920eb))
* [x] your code should contain comments to describe the problem (*e.g.* what expected and actually happened?)

Hi @GuangchuangYu, thanks for `ggtree`! There is some behavior that I think may be incorrect. I hope you can fix that, or my misunderstanding :+1:. Cheers, @richelbilderbeek  

Here is the reprex:

```{r}
library(ape)
library(ggplot2)

# Here I create two trees. In one tree, taxon 'C' is the outlier. In the other tree, taxon 'B' is the outlier
tree_1 <- ape::read.tree(text = "((A:2,B:2):1, C:3);")
tree_2 <- ape::read.tree(text = "((A:1,C:1):2, B:3);")

# Show it in a picture:
plot(tree_1)
plot(tree_2)
```

![tree_1](https://user-images.githubusercontent.com/2098230/51738822-57af5a80-2090-11e9-9ca0-35c5d770c4f1.png)

![tree_2](https://user-images.githubusercontent.com/2098230/51738823-57af5a80-2090-11e9-9a75-24b1495cbd09.png)

```
# Combine the trees
trees <- c(tree_1, tree_2)

# Show them overlapping
ggtree::ggtree(trees) + ggtree::geom_tiplab()
```

The tree created by `ggtree` shows that C is the outlier in both cases:

![tree_created](https://user-images.githubusercontent.com/2098230/51738836-5f6eff00-2090-11e9-9bdf-9ca460f335c9.png)

This is a plot I would have expected, in which B and C are outliers each once:

![tree_expected](https://user-images.githubusercontent.com/2098230/51738837-5f6eff00-2090-11e9-80ba-f4a37fd2164a.png)

### Ask in right place

* [x] for bugs or feature requests, post here (github issue)
* [ ] for questions, please post to [google group](https://groups.google.com/forum/#!forum/bioc-ggtree)

**Comment by GuangchuangYu** [comment]
Yes, it is a bug. This bug was caused by supporting `phyloseq` object, we need to allow node duplications, see <https://github.com/GuangchuangYu/ggtree/blob/master/R/method-fortify.R#L185>.

So that I have a [setup_tree_data](https://github.com/GuangchuangYu/ggtree/blob/5aa9cec45cf99a41eac2bf693ece2b536ea17f24/R/geom_tree.R#L165-L171) function to keep non-duplicated one for drawing branches and labels, etc.

So, in your case, only tip labels of tree_1 were plotted.

If I fixed this bug, then I should remove the support of `phyloseq`.

I will think about it to figure out a better way. Don't hesitate to let me know, if you have ideas.

**Comment by brj1** [comment]
It now works in my fork. See #222
